.github/WHAT_IS_MAKER.md

```diff
@@ -0,0 +1,63 @@
+# What is Maker?
+
+Most people contributing to and using Maker will be familiar with [Orbit](https://github.com/square/orbit) and will likely have a lot of wrong ideas about Maker's goals by assuming they're the same as Orbit's. This is incorrect. Maker is not Orbit. Their key high-level differences in a table:
+
+| | Orbit | Maker |
+|-|-------|-------|
+| **Visual Design** | <ul><li>Desktop-first design, mobile was an afterthought or not considered at all</li><li>Designed for information-dense dashboards</li><li>Not themeable, specifically designed to reflect Weebly's and Square's brand</li></ul> | <ul><li>Mobile-first design</li><li>Designed for ecommerce sites</li><li>Designed to be themeable, should be able to be customized to fit any Seller's brand</li></ul> |
+| **API Design** | <ul><li>High-level, opinionated, batteries-included components</li><li>Many inter-component dependencies, e.g. OButton is reused in the public API of a dozen other components</li><li>Doesn't export components for low-level concerns like transitions & animations</li><li>Icon set bundled with library</li></ul> | <ul><li>Mid-level, modular, compose-these-parts-together-yourself components</li><li>Components are standalone, there's no inter-component dependencies (at least not in the public API)</li><li>Exports components for low-level concerns like transitions & animations</li><li>Bring your own icons</li></ul> |
+| **Maintanence** | <ul><li>Maintained by a single team</li><li>Only one team had the authority to approve & publish changes</li></ul> | <ul><li>Collectively maintained by all teams</li><li>All teams have authority to approve & publish changes (which includes taking responsiblity for breaking other teams' code if that becomes the case)</li></ul> |
+| **Versioning** | <ul><li>Follows semver conventions</li><li>Infrequent but large major releases with many breaking changes across many components</li><li>Rarely publishs pre-releases</li></ul> | <ul><li>Follows semver conventions</li><li>Frequent but small major releases with breaking changes typically scoped to a single component</li><li>Regularly publishs pre-releases to experiment with and stabilize eventual breaking API changes before making official releases</li></ul> |
+
+
+
+## FAQs
+
+
```

Comment:
Can you address the question: 

> Should I use maker?

If I'm a product engineer, should I use it? Or should I hold off till I'm explicitly told to do so from my designer?